src/utils.py

```diff
@@ -29,25 +35,33 @@ def run_policy_on_task(policy: Callable[[State], Action], task: Task,
     atoms = abstract(state, predicates)
     states = [state]
     actions: List[Action] = []
+    video: Video = []
+    if make_video:
+        assert render is not None
+        render = cast(Callable[[State], Image], render)
```

Comment:
I think that since you have the assert right above this, you won't need this cast (mypy would be smart enough to realize that the "Optional" should get dropped since it was asserted)